frontend/src/containers/PublicGroup.js

```diff
@@ -52,155 +48,72 @@ export class PublicGroup extends Component {
-
-    const emptyState = (
-      <div className='PublicGroup-emptyState'>
-        <div className='PublicGroup-donationIcon'>
-          <Icon type='revenue' />
-        </div>
-        <label>
-          All your latest donations will show up here
-        </label>
-      </div>
-    );
-
-    return (
-      <div className='PublicGroup-donations'>
-        <h2>Revenue</h2>
-        {(donations.length === 0) && emptyState}
-
-        {donations.map(donation => <TransactionItem key={donation.id} transaction={donation} user={users[donation.UserId]} />)}
-
-        {donations.length >= NUM_TRANSACTIONS_TO_SHOW && (
-          <Link className='PublicGroup-tx-link' to={`/${group.slug}/donations`}>
-            See all donations
-          </Link>
-        )}
-      </div>
-    );
+  _closeDonationFlow() {
+    this.setState({
+      showThankYouMessage: false,
+      showUserForm: false
+    });
   }
 
   render() {
     const {
       group,
-      shareUrl,
+      expenses,
+      donations,
+      users,
+      // shareUrl,
     } = this.props;
 
     return (
       <div className='PublicGroup'>
-
-        <PublicTopBar />
         <Notification />
 
-        <div className='PublicContent'>
+        <PublicGroupHero group={group} />
+        <PublicGroupWhoWeAre group={group} />
+        <PublicGroupWhyJoin group={group} expenses={expenses} />
 
-          <div className='PublicGroupHeader'>
-            <img className='PublicGroupHeader-logo' src={group.logo ? group.logo : '/static/images/media-placeholder.svg'} />
-            <div className='PublicGroupHeader-website'><DisplayUrl url={group.website} /></div>
-            <div className='PublicGroupHeader-host'>Hosted by <a href={group.host.website}>{group.host.name}</a></div>
-            <div className='PublicGroupHeader-description'>
-              {group.description}
-            </div>
-          </div>
-
-          <Media group={group} />
-
-          <div className='PublicGroup-summary'>
-            <div className='PublicGroup-metricContainer'>
-              <Metric
-                label='Funds Available'
-                value={formatCurrency(group.balance, group.currency, {precision: 0})} />
-              <Metric
-                label='Backers'
-                value={group.backersCount} />
-            </div>
-            <a className='Button Button--green PublicGroup-support' href='#support'>
-              Back us
-            </a>
-            <div className='PublicGroup-share'>
-              <ShareIcon type='twitter' url={shareUrl} name={group.name} description={group.description} />
-              <ShareIcon type='facebook' url={shareUrl} name={group.name} description={group.description} />
-              <ShareIcon type='mail' url={shareUrl} name={group.name} description={group.description} />
-            </div>
-          </div>
+        <div className='bg-light-gray px2'>
+          <PublicGroupJoinUs {...this.props} donateToGroup={donateToGroup} />
```

Comment:
Replace with `donateToGroup={donateToGroup.bind(this)}` or you will get a `this.refreshData is not a function` error